### ISSUE TITLE
how create the trafic.rar using traffic from TOMTOM or here

### ISSUE THREAD

**Issue by Jofemago** [valhalla/valhalla#4915]
### Discussed in https://github.com/valhalla/valhalla/discussions/4256

<div type='discussions-op-text'>

<sup>Originally posted by **Jofemago** August 15, 2023</sup>
Hello . how are you? I'm trying to configure traffic in Valhalla for Colombia, and I have many doubts about traffic. I'm working with the TAR file, and I'm looking at TomTom's documentation. I can see that I can obtain flow information, but it's a bit confusing for me. It doesn't cover the whole city, just a part with a radius around a point. On the other hand, I can obtain traffic data using a polygon with HERE, but I find it quite confusing to define what should go inside that TAR file. Do you have any documentation that could provide me with more clarity? Thank you very much.</div>

**Comment by khalidDaoud** [comment]
It doesn't seem like a straight forward thing, check below article 
https://gis-ops.com/traffic-in-valhalla/

**Comment by chrstnbwnkl** [comment]
Yes, converting traffic data from a third party vendor is not trivial. I can offer help, feel free to email me.
